build.gradle

```diff
@@ -199,8 +199,10 @@ dependencies {
             force("org.apache.logging.log4j:log4j-api:${log4jVersion}")
             force("org.apache.logging.log4j:log4j-core:${log4jVersion}")
             force("org.apache.logging.log4j:log4j-jul:${log4jVersion}")
-            force("org.slf4j:slf4j-api:${slf4jVersion}")
+            //force("org.slf4j:slf4j-api:${slf4jVersion}")
         }
+
+        exclude group: 'org.slf4j'
```

Comment:
Is this a transitive dependency coming from some other dependency?

Reply:
Comment addressed and verified by dan

Reply:
@owaiskazi19 this turned out to be a gnarlier issue than I thought.  I'll update the issue and create a new one.

TLDR we are keeping both imports as is.  We are just moving the log4j xml file.